src/registerToolbar.ts

```diff
@@ -56,6 +51,38 @@ export default function registerToolbar() {
         type: ToolbarItemType.State,
     });
 
+    ToolbarItemFactory.register(CopyRegions, {
```

Comment:
Right now there are basically 2 categories of toolbar buttons... 1 where they have code that perform something onclick, and the other where their type is inspected in the editor page to dispatch operations.  For these they appear to do the later, but I think there is some code bloat going on where we can simplify this by creating a new `GenericToolbarButton` that doesn't really do anything and avoids us having to create these stub button components.